### ISSUE TITLE
Questions about Macro Hidden States 

### ISSUE THREAD

**Issue by quantmacroeco** [LouisChen1992/Deep_Learning_in_Asset_Pricing#6]
Would you mind provide little details about how to get the hidden states using lstm model? For example ,what loss function did you use?

**Comment by ChadyRaach** [comment]
@quantmacroeco I think that the two networks (SDF Network and Conditional network) are trained end-to-end. This means that the LSTM of each of the two models will be trained using the same backpropagation process as their corresponding network.
I quote the paper: 

> The two LSTMs are based on the criteria function of the two networks, that is h_t are the hidden states that can minimize the pricing errors, while h^g_t generate the test assets with the largest mispricing
